src/TiledArray/math/solvers/cp/cp.h

```diff
@@ -140,10 +190,14 @@ class CP {
     return epsilon;
   }
 
-  std::vector<Array> get_factor_matrices() {
+  std::vector<Array> get_factor_matrices(bool with_lambda = false) {
     TA_ASSERT(!cp_factors.empty(),
               "CP factor matrices have not been computed)");
     auto result = cp_factors;
+    if(with_lambda){
```

Comment:
@kmp5VT you don't seem to have git hooks installed, please see https://valeevgroup.github.io/tiledarray/dox-master/_workflow-_notes.html